SemanticCompoundQueries.php

```diff
@@ -17,7 +17,7 @@
 
 if ( !defined( 'MEDIAWIKI' ) ) die();
 
-define( 'SCQ_VERSION', '0.4.1' );
+define( 'SCQ_VERSION', '1.0' );
```

Comment:
I don't agree that one should only go to 1.0 once *internal* quality reaches a certain threshold. SemVer wise it's about being ready to be used in production, which this extension has been for years.